db/schema.rb

```diff
@@ -69,6 +69,7 @@
     t.index ["game_id"], name: "index_pieces_on_game_id"
   end
 
+
```

Comment:
Please delete the new line. We shouldn't mess with schema.rb unless there is a migration to go with the change. (previous experience with this, things can get messed up)